src/lib.rs

```diff
@@ -130,3 +159,111 @@ nonmax!(NonMaxU32, NonZeroU32, u32);
 nonmax!(NonMaxU64, NonZeroU64, u64);
 nonmax!(NonMaxU128, NonZeroU128, u128);
 nonmax!(NonMaxUsize, NonZeroUsize, usize);
+
+// https://doc.rust-lang.org/stable/src/core/convert/num.rs.html#383-407
```

Comment:
```suggestion
// https://doc.rust-lang.org/1.47.0/src/core/convert/num.rs.html#383-407
```

Links to sources from `stable` aren't permanent since stable is a moving target. Let's pin these to the latest stable version!